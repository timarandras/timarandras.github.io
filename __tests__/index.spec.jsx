import { render } from '@testing-library/react';
import { useStaticQuery } from 'gatsby';
import React from 'react';

import Index from '../src/pages';

describe('Index', () => {
  test('Matches snapshot', () => {
    useStaticQuery.mockReturnValueOnce({ site: { siteMetadata: { description: 'Dummy site' } } });
    const { container } = render(<Index />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
