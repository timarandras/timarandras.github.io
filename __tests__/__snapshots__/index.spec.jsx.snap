// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthMd"
>
  <div>
    content
  </div>
</div>
`;
